src/StudioComponents/VerticalCollapsibleSection.lua

```diff
@@ -52,30 +52,30 @@ return function(props: VerticalExpandingListProperties): Frame
 		elseif isHovering then
 			return Enum.StudioStyleGuideModifier.Hover
 		end
-		return Enum.StudioStyleGuideModifier.Default
+		return themeProvider.IsDark:get() and Enum.StudioStyleGuideModifier.Default or Enum.StudioStyleGuideModifier.Pressed
```

Comment:
Like one of the previous comments, unwrap(themeProvider.IsDark) should be a variable defined at the top of the computed since it might not get captured.

It's also preferable to use an if then else statement in a case like this instead of an and or statement.

Reply:
Like I said last time, the "might not get captured" part is a feature, not a bug. We don't want to capture a dependency that isn't impacting the result, as that would be causing pointless recomputing.

I'll switch to the new if-then, old habits die hard!